segments/helpers.py

```diff
@@ -173,10 +173,13 @@ def execute_raw_user_query(sql):
     if sql is None or not type(sql) == str or 'select' not in sql.lower():
         return
 
-    with connections[app_settings.SEGMENTS_EXEC_CONNECTION].chunked_cursor() as cursor:
+    with connections[app_settings.SEGMENTS_EXEC_CONNECTION].cursor() as cursor:
```

Comment:
Can we note why this wasn't working in the PR?

Reply:
@sramirezgroveco It failed with `cursors can only be used within the transaction that created them` error on redshift, while was working fine on postgres. I've tried to wrap it is transaction.atomic() block, but got same error. Then I stopped digging into server side cursors and used `fetchmany` which works fine both on postgres and redshift and likely on other engines like mysql or sqllite .